### ISSUE TITLE
Pause button for autoplay videos

### ISSUE THREAD

**Issue by cogniSyb** [Netcentric/vg-volvotrucks-us-rd#156]
Blocks with an autoplay video should have a pause button, to comply with accessibility standards. Specifically, we’re mitigating [Failure 93 of Success Criterion 1.4.2](https://www.w3.org/TR/WCAG20-TECHS/F93.html).

Designs have been adjusted to show a pause button. For now, we’re focusing on the hero #41, [the design is available](https://www.figma.com/file/yv0VtHTvMoGPqkv0RAYcVp/PDP?type=design&node-id=6568-48961&mode=design&t=Rr7XpNdhodaMcgfL-4). 

- The video player for autoplay videos should have a visible pause button when the video starts playing
- The button must be discernible from other controls and be clearly labeled and icon that indicates its functionality
- The pause/play button should be accessible to users navigating via keyboard, meaning it should be focusable and actionable using keyboard inputs. The focus style should be the same as for the scroll to top button
- Upon pressing the pause button, the autoplay video should immediately stop playing
- Once the video is paused, the [pause button should change to a play button](https://www.figma.com/file/Id3nd9kbfh8TpS1SfWhrW2/Volvo-Component-Library?type=design&node-id=566-2770&mode=dev)
- The button should be large enough to be easily tapped on touch devices without accidentally pressing adjacent controls

**Comment by DanielaPedrochevd** [comment]
@Joesturbo This issue is a blocker for the MVP of the 9th of October, it will probably mean we will have to postpone the go-live date to the 11th.

**Comment by DanielaPedrochevd** [comment]
@markovukiceviccn is this done?

**Comment by markovukiceviccn** [comment]
@DanielaPedrochevd @Andrei-Boiangiu 
Somehow, I missed this convo.
I will be checking it immediately.

**Comment by markovukiceviccn** [comment]
@Andrei-Boiangiu @DanielaPedrochevd @#198
This is not a bug, take a look and if you need any further clarification feel free to ping me.
![Screenshot_2023-10-13_at_10_47_55](https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/139957716/b606e707-470d-4055-a7e6-818e8398bbcb)

**Comment by markovukiceviccn** [comment]
@Andrei-Boiangiu @DanielaPedrochevd 

This will be fixed in https://github.com/Netcentric/vg-volvotrucks-us-rd/issues/182
